profiles/realtime-virtual-host/script.sh

```diff
@@ -2,37 +2,39 @@
 
 . /usr/lib/tuned/functions
 
-KTIMER_LOCKLESS_FILE=/sys/kernel/ktimer_lockless_check
-
 start() {
     setup_kvm_mod_low_latency
-
-    disable_ksm
-
-    if [ -f $KTIMER_LOCKLESS_FILE ]; then
-        echo 1 > $KTIMER_LOCKLESS_FILE
-    fi
-
     return 0
 }
 
 stop() {
     if [ "$1" = "full_rollback" ]; then
         teardown_kvm_mod_low_latency
-        enable_ksm
     fi
     return "$?"
 }
 
 verify() {
+    # set via /etc/modprobe.d/kvm.conf and /etc/modprobe.d/kvm.rt.tuned.conf
```

Comment:
I think retval should be explicitly set to '0' here, because otherwise you could return it on line 38 expanded to empty string, which will have the same effect as returning '0', but it may look suspicious.

Reply:
done.